File: src/main/java/instance/common/ShutDown.java
package instance.common;

import se.sics.kompics.address.Address;
import se.sics.kompics.network.Message;

import java.io.Serializable;

public class ShutDown extends Message implements Serializable {

	private static final long serialVersionUID = -5909108939372221250L;

	public ShutDown(Address source, Address destination) {
		super(source, destination);
	}


}

File: src/main/java/instance/common/ShutDown.scala
package instance.common

import se.sics.kompics.network.Message
import se.sics.kompics.address.Address

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class ShutDown(source:Address, destination:Address) extends Message(source, destination)